behave_django/environment.py

```diff
@@ -49,3 +67,11 @@ def run_hook(self, name, context, *args):
             django_test_runner.after_scenario(context)
 
     ModelRunner.run_hook = run_hook
+
+
+def before_scenario(*args, **kwargs):
+    warnings.warn("DEPRECATED: This function is no longer needed.")
```

Comment:
`stacklevel=2` could be nice to prevent misunderstandings. (Not *this* method is no longer needed, but the one calling it!) -- Same below.